Compilation crashes on MacOS

```bash
 Nim Output /Users/username/.cache/nim/nimlangserver_r/@m..@s..@spkgs2@schronos-4.0.3-fd6062c492e5b6053702638159d38d1219b4bf72@schronos@sstreams@stlsstream.nim.c:2360:58: error: incompatible pointer types passing 'NU *' (aka 'unsigned long long *') to parameter of type 'size_t *' (aka 'unsigned long *') [-Wincompatible-pointer-types]
        ...  2360 |                         (*colonenvP_).buf11 = br_ssl_engine_sendrec_buf(T28_, (&(*colonenvP_).length10));
        ...       |                                                                               ^~~~~~~~~~~~~~~~~~~~~~~~~
        ... /Users/username/.nimble/pkgs2/bearssl-0.2.5-550e6f9321b85de53bba9c0ffab9c95ffbe12ab3/bearssl/abi/../csources/inc/bearssl_ssl.h:2144:43: note: passing argument to parameter 'len' here
        ...  2144 |         const br_ssl_engine_context *cc, size_t *len);
        ...       |                                                  ^
        ... /Users/username/.cache/nim/nimlangserver_r/@m..@s..@spkgs2@schronos-4.0.3-fd6062c492e5b6053702638159d38d1219b4bf72@schronos@sstreams@stlsstream.nim.c:2983:59: error: incompatible pointer types passing 'NU *' (aka 'unsigned long long *') to parameter of type 'size_t *' (aka 'unsigned long *') [-Wincompatible-pointer-types]
        ...  2983 |                                 (*colonenvP_).buf12 = br_ssl_engine_sendapp_buf(T61_, (&(*colonenvP_).length11));
        ...       |                                                                                       ^~~~~~~~~~~~~~~~~~~~~~~~~                                                                             ... /Users/username/.nimble/pkgs2/bearssl-0.2.5-550e6f9321b85de53bba9c0ffab9c95ffbe12ab3/bearssl/abi/../csources/inc/bearssl_ssl.h:2086:43: note: passing argument to parameter 'len' here               ...  2086 |         const br_ssl_engine_context *cc, size_t *len);                                  ...       |                                                  ^
        ... /Users/username/.cache/nim/nimlangserver_r/@m..@s..@spkgs2@schronos-4.0.3-fd6062c492e5b6053702638159d38d1219b4bf72@schronos@sstreams@stlsstream.nim.c:3495:58: error: incompatible pointer types passing 'NU *' (aka 'unsigned long long *') to parameter of type 'size_t *' (aka 'unsigned long *') [-Wincompatible-pointer-types]                                                                          ...  3495 |                         (*colonenvP_).buf11 = br_ssl_engine_recvrec_buf(T28_, (&(*colonenvP_).length10));                                                                                   ...       |                                                                               ^~~~~~~~~~~~~~~~~~~~~~~~~                                                                                     ... /Users/username/.nimble/pkgs2/bearssl-0.2.5-550e6f9321b85de53bba9c0ffab9c95ffbe12ab3/bearssl/abi/../csources/inc/bearssl_ssl.h:2173:43: note: passing argument to parameter 'len' here
        ...  2173 |         const br_ssl_engine_context *cc, size_t *len);                                  ...       |                                                  ^
        ... fatal error: too many errors emitted, stopping now [-ferror-limit=]                             ... 4 errors generated.
        ... Error: execution of an external compiler program 'clang -c  -w -ferror-limit=3 -pthread -I/Users/username/.nimble/pkgs2/bearssl-0.2.5-550e6f9321b85de53bba9c0ffab9c95ffbe12ab3/bearssl/abi -I/Users/username/.nimble/pkgs2/bearssl-0.2.5-550e6f9321b85de53bba9c0ffab9c95ffbe12ab3/bearssl/abi/../csources/src/ -I/Users/username/.nimble/pkgs2/bearssl-0.2.5-550e6f9321b85de53bba9c0ffab9c95ffbe12ab3/bearssl/abi/../csources/inc/ -I/Users/username/.nimble/pkgs2/bearssl-0.2.5-550e6f9321b85de53bba9c0ffab9c95ffbe12ab3/bearssl/abi/../csources/tools/ -DBR_USE_UNIX_TIME=1 -DBR_USE_URANDOM=1 -DBR_LE_UNALIGNED=1 -DBR_64=1 -DBR_INT128=1 -I/Users/username/.nimble/pkgs2/zlib-0.1.0-7033fe964f174dccd20dcbd71a8150e30e5651fb/zlib/csources -DHAVE_UNISTD_H -O3   -I/Users/username/.nimble/pkgs2/nim-2.0.8-83c1d893cb997417565b7208e3cbebb8f93222cb/lib -I/Users/username/.nimble/buildtemp/nimlangserver-1.12.0-0f0f7f3ceb0245de876bc9a58dc7015160d739fd -o /Users/username/.cache/nim/nimlangserver_r/@m..@s..@spkgs2@schronos-4.0.3-fd6062c492e5b6053702638159d38d1219b4bf72@schronos@sstreams@stlsstream.nim.c.o /Users/username/.cache/nim/nimlangserver_r/@m..@s..@spkgs2@schronos-4.0.3-fd6062c492e5b6053702638159d38d1219b4bf72@schronos@sstreams@stlsstream.nim.c' failed with exit code: 1
        ...                                                                                                Tip: 66 messages have been suppressed, use --verbose to show them.
vnext.nim(1128)          buildFromDir
                                                                   Error:  Build failed for the package: nimlangserver
```

--- a/nimlangserver.nims
+++ b/nimlangserver.nims
@@ -1,0 +1,2 @@
+if defined(clang):
+  switch("passC", "-Wno-incompatible-pointer-types")

